XenModel/XenAPI-Extensions/Host.cs

```diff
@@ -178,7 +178,7 @@ public override string ToString()
         public override string Description()
         {
             if (name_description == "Default install of XenServer" || name_description == "Default install") // i18n: CA-30372, CA-207273
-                return Messages.DEFAULT_INSTALL_OF_XENSERVER;
+                return string.Format(Messages.DEFAULT_INSTALL_OF_XENSERVER, software_version["product_brand"]);

```

Comment:
To be on the safe side, we should check the key "product_brand" exists in software_version before calling `software_version["product_brand"]`. If the key doesn't exist, we can default to Branding.PRODUCT_BRAND

Reply:
Reasonable. But I would prefer Messages.XENSERVER instead of Branding.PRODUCT_BRAND, as Branding is in XenAdmin and Host is in XenModel.